hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -959,22 +936,23 @@
       ]
     }
 
-    { name: "wake_config",
-      desc: "USB wake configuration for suspend / resume",
-      swaccess: "rw",
+    { name: "wake_control",
+      desc: "USB wake module control for suspend / resume",
+      swaccess: "wo",
       hwaccess: "hro",
+      hwqe: "true",
       async: "clk_aon_i",
       fields: [
         {
           bits: "0",
           resval: "0",
-          name: "wake_en",
+          name: "suspend_req",
           desc: '''
-                Enable the usb resume wake function.  When this is set, a resume indication from
-                a usb host can be used to drive a wake from sleep event.
+                Suspend request to the wake detection module.
 
-                Note this function is meant to be set as a mode and not toggled on/off every time
```

Comment:
I have changed this from a mode bit to a trigger, putting software entirely in control of sequencing and simplifying the hardware logic a little. However, this may increase the time to reach low power by tens of microseconds (out of a budget of 6-7 ms), due to the additional CSR activity in the AON domain. Let me know if there are strong feelings for it to be the other way.

Reply:
o wait i see the module active below.